src/components/filters/useLastSeenFilter.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { LAST_SEEN_CHIP, lastSeenItems } from '../../Utilities/constants';
+
+export const lastSeenFilterState = { lastSeenFilter: [] };
+export const LAST_SEEN_FILTER = 'LAST_SEEN_FILTER';
+export const lastSeenFilterReducer = (_state, { type, payload }) => ({
+    ...type === LAST_SEEN_FILTER && {
+        lastSeenFilter: payload
+    }
+});
+
+export const uselastSeenFilter = ([state, dispatch] = [lastSeenFilterState]) => {
+    let [lastSeenStateValue, setLastSeenValue] = useState({});
+    const lastSeenValue = dispatch ? state.lastSeenFilter : [lastSeenStateValue];
+    const setValue = dispatch ? (newValue) => dispatch({ type: LAST_SEEN_FILTER, payload: newValue }) : setLastSeenValue;
+
+    const filter = {
+        label: 'Last seen',
+        value: 'last_seen',
+        type: 'radio',
```

Comment:
I believe this is not really important, but the mocks don't have radio buttons:

![image](https://user-images.githubusercontent.com/31385370/223110398-2f2546a3-bbaf-408b-b17a-a8386d00b5be.png)

if it doesn't require big changes, I think we can align it more to the mock's look

Reply:
The ticket hasnt been updated, but there was a conversation with Margot from UX that gives the green light for radio-buttons to align more with the other apps!